funnel/templates/503.html.jinja2

```diff
@@ -0,0 +1,13 @@
+{% extends "layout.html.jinja2" %}
+{% block title %}503 Service Unavailable{% endblock %}
+
+{% block content %}
+  <div>
+    <img src="{{ url_for('static', filename='img/error-503.svg') }}" class="emoticon"/>
+    <p class="mui--text-light mui--text-subhead">{% trans %}Yikes! Server went away{% endtrans %}</p>
```

Comment:
I'm not comfortable with this language in a critical place as this. 503 typically means (a) server overload, (b) downtime for maintenance and upgrades. This page is critical for explaining to a user that they should wait or check the status website for updates.